views/index.js

```diff
@@ -0,0 +1,37 @@
+
+fetch("http://news-summary-api.herokuapp.com/guardian?apiRequestUrl=http://content.guardianapis.com/search?show-fields=thumbnail&show-elements=image")
+  .then(response => {
+    console.log("response:")
+    console.log(response)
```

Comment:
I would delete few of these console log statements to make the code more readable